ppr-ui/src/interfaces/mhr-registration-interfaces/MhrRegistrationIF.ts

```diff
@@ -48,3 +48,49 @@ export interface MhrRegistrationIF {
     }
   ]
 }
+
+export interface NewMhrRegistrationApiIF {
```

Comment:
How close is this to our UI equivalent IF?

Reply:
pretty close, mainly just excluding `owners` and the required properties are different

Reply:
Cool, thanks!